requirements/base.txt

```diff
@@ -64,7 +74,7 @@ dnspython==2.1.0
     # via email-validator
 email-validator==1.1.3
     # via flask-appbuilder
-flask==1.1.4
+flask==2.0.3
```

Comment:
Per the [changelog](https://flask.palletsprojects.com/en/2.0.x/changes/#version-2-0-0) the only breaking change which _may_ impact us is https://github.com/pallets/flask/issues/3555.